File: tests/corpus/clean/slop_clean.clj
(ns corpus.slop-clean)

(def domain-limit 37)

(defn uses-named-limit [input]
  (+ input domain-limit))

(defn complete-small-function [input]
  (+ input 1))

(defn documented-reason [input]
  ; Domain rule: retain the raw value for downstream reconciliation.
  input)

File: tests/corpus/clean/slop_clean.jl
module SlopCleanJulia

const DOMAIN_LIMIT = 37

function uses_named_limit(input)
    input + DOMAIN_LIMIT
end

function complete_small_function(input)
    input + 1
end

function documented_reason(input)
    # Domain rule: retain the raw value for reconciliation.
    input
end

end

